packages/@sanity/base/src/datastores/user/createUserStore.ts

```diff
@@ -115,7 +115,7 @@ function arrayify(users: User | User[]): User[] {
 }
 
 function isUser(thing: unknown): thing is User {
-  return thing && thing !== null && typeof (thing as User).id === 'string'
+  return thing !== null && typeof (thing as User).id === 'string'
```

Comment:
If `thing` is undefined, this will crash. Maybe we can cast the expression to boolean instead?
```suggestion
  return Boolean(thing && thing !== null && typeof (thing as User).id === 'string')
```

Reply:
Good point! Added in d8dab71da 🙂